app/app/settings/base.py

```diff
@@ -43,8 +51,18 @@
     "django.middleware.locale.LocaleMiddleware",
     "django.middleware.gzip.GZipMiddleware",
     "django_htmx.middleware.HtmxMiddleware",
+    # AxesMiddleware should be the last middleware in the MIDDLEWARE list.
+    "axes.middleware.AxesMiddleware",
 ]
 
+# Axes configuration
+AXES_FAILURE_LIMIT = config("AXES_FAILURE_LIMIT", default=5, cast=int)
+AXES_COOLOFF_TIME = config("AXES_COOL_OFF_TIME", default=0.01, cast=float)  # in hours
```

Comment:
I think thirty six seconds is too short. Let's make it half an hour by default (people can always change it)

```suggestion
AXES_COOLOFF_TIME = config("AXES_COOL_OFF_TIME", default=0.5, cast=float)  # in hours
```

Reply:
Yes, 36 seconds is way too short. I initially had it at 15 mins but I must've accidentally changed it during testing.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/856/commits/94d8aa86bfc39e7278bc2f35ad0838df8734eb62